app/Rules/MaxResultsPerPageRule.php

```diff
@@ -36,11 +37,7 @@ public function passes($attribute, $value): bool
 
     public function message(): array|string
     {
-        return "Value {$this->value} is higher than the configured '{$this->maxResultsPerPage()}' max value.";
```

Comment:
Will this be thrown as an API response error?

Reply:
This is a request validation rule, which prevents users to specify higher value for the `limit` parameter than the configured value in the configuration.